### ISSUE TITLE
[FE] My VA: Update Cerner alert header

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#24905]
The h3 should be an h2

![Screen Shot 2021-05-18 at 3.10.13 PM.png](https://images.zenhubusercontent.com/5b4d0309a8d8be5bda39303e/ad00fe8f-9066-4e09-abba-d17c34cc3124)

It looks like the other headers on the alert will still work when the top-level is switched to an h2.

**Comment by erikphansen** [comment]
This is ready to validate on staging
